terraform/aws/modules/atlantis/chart_values.yaml

```diff
@@ -69,58 +68,58 @@ repoConfig: |
    core:
      plan:
        steps:
-       - run: export AWS_PROFILE="AWS_CORE"
-       - init
-       - plan
+       - run: "cd /home/atlantis/ && [ ! -f 'terraform' ] && curl https://releases.hashicorp.com/terraform/0.13.5/terraform_0.13.5_linux_amd64.zip --output tf.zip && unzip tf.zip"
+       - run: AWS_PROFILE="AWS_CORE" terraform init
+       - run: AWS_PROFILE="AWS_CORE" terraform plan --out=/tmp/plan_core
      apply:
        steps:
-       - run: export AWS_PROFILE="AWS_CORE"
-       - apply
+       - run: "cd /home/atlantis/ && [ ! -f 'terraform' ] && curl https://releases.hashicorp.com/terraform/0.13.5/terraform_0.13.5_linux_amd64.zip --output tf.zip && unzip tf.zip"
+       - run: AWS_PROFILE="AWS_CORE" terraform apply --out=/tmp/plan_core
    dev:
      plan:
        steps:
-       - run: export AWS_PROFILE="AWS_DEV"
-       - init
-       - plan
+       - run: "cd /home/atlantis/ && [ ! -f 'terraform' ] && curl https://releases.hashicorp.com/terraform/0.13.5/terraform_0.13.5_linux_amd64.zip --output tf.zip && unzip tf.zip"
+       - run: AWS_PROFILE="AWS_DEV" terraform init
+       - run: AWS_PROFILE="AWS_DEV" terraform plan --out=/tmp/plan_dev
      apply:
        steps:
-       - run: export AWS_PROFILE="AWS_DEV"
-       - apply
+       - run: "cd /home/atlantis/ && [ ! -f 'terraform' ] && curl https://releases.hashicorp.com/terraform/0.13.5/terraform_0.13.5_linux_amd64.zip --output tf.zip && unzip tf.zip"
+       - run: AWS_PROFILE="AWS_DEV" terraform apply --out=/tmp/plan_dev
    test:
      plan:
        steps:
-       - run: export AWS_PROFILE="AWS_TEST"
-       - init
-       - plan
+       - run: "cd /home/atlantis/ && [ ! -f 'terraform' ] && curl https://releases.hashicorp.com/terraform/0.13.5/terraform_0.13.5_linux_amd64.zip --output tf.zip && unzip tf.zip"
+       - run: AWS_PROFILE="AWS_TEST" /home/atlantis/terraform init
+       - run: AWS_PROFILE="AWS_TEST" /home/atlantis/terraform plan --out=/tmp/plan_test
      apply:
        steps:
-       - run: export AWS_PROFILE="AWS_TEST"
-       - apply
+       - run: "cd /home/atlantis/ && [ ! -f 'terraform' ] && curl https://releases.hashicorp.com/terraform/0.13.5/terraform_0.13.5_linux_amd64.zip --output tf.zip && unzip tf.zip"
+       - run: AWS_PROFILE="AWS_TEST" terraform apply --out=/tmp/plan_test
    staging:
      plan:
        steps:
-       - run: export AWS_PROFILE="AWS_STAGING"
-       - init
-       - plan
+       - run: "cd /home/atlantis/ && [ ! -f 'terraform' ] && curl https://releases.hashicorp.com/terraform/0.13.5/terraform_0.13.5_linux_amd64.zip --output tf.zip && unzip tf.zip"
```

Comment:
@stylianosrigas @cpanato Just FYI: this is a temporary hack/workaround to make it work until the upstream application's repository releases the proper fix. [ref.](https://github.com/runatlantis/atlantis/issues/1153#issuecomment-748185839)